server/utils/__tests__/jsonApi.spec.js

```diff
@@ -557,3 +558,76 @@ describe('isNew', () => {
     expect(isNew('NOT_A_VALID_DATE')).toBe(false);
   });
 });
+
+describe('cropTextWithEllipsis', () => {
+  let item;
+  let outputItem;
+
+  beforeEach(() => {
+    item = {
+      id: 999999,
+      contentType: 'page',
+      externalContent: false,
+      title:
+        'A title that is long enough to be cropped with an ellipse added to the end',
+      summary: 'A summary',
+      contentUrl: '/content/999999',
+      displayUrl: undefined,
+      image: {
+        url: 'url path',
+        alt: 'Alt text',
+      },
+      isNew: false,
+    };
+
+    outputItem = {
+      id: 999999,
+      contentType: 'page',
+      externalContent: false,
+      title: 'A title that is long enough...',
+      summary: 'A summary',
+      contentUrl: '/content/999999',
+      displayUrl: undefined,
+      image: {
+        url: 'url path',
+        alt: 'Alt text',
+      },
+      isNew: false,
+    };
```

Comment:
Fair bit of duplication. I'd have been tempted to cheat a bit:
```
outputItem = {
  ...item,
  title: 'A title that is long enough...',
}
```

Reply:
good catch.

Reply:
updated